assets/css/abstracts/_mixins.scss

```diff
@@ -1,10 +1,14 @@
-// Converts a font-size in px to rem and optionally sets a relative line-height.
-@mixin font-size($sizeValue: 16px, $lineHeight: false) {
-	font-size: ($sizeValue / 16px) * 1rem;
-
-	@if ($lineHeight) {
-		line-height: $lineHeight / $sizeValue;
-	}
+$fontSizes: (
+	"smaller": 0.75,
+	"small": 0.875,
+	"regular": 1,
+	"large": 1.25,
+	"larger": 2,
+);
+
+// Maps a named font-size to its em equivalent.
+@mixin font-size($sizeName) {
+	font-size: map-get($fontSizes, $sizeName) * 1em;
 }
```

Comment:
We saw `rem` units were not playing well with many themes because they set `<html>` font size to something very small and then make it bigger in the `<body>`. One example of this is Twenty Twenty:
https://github.com/WordPress/twentytwenty/blob/master/style.css#L117

In pb0Spc-Cv-p2 I proposed switching to `em` to fix this. Considering `rem` can give so different results depending on the theme, I think we should avoid it when possible.

Reply:
👍 EM is relative to the current content too so it should fit better within themes. Not all themes use the html/body class for the base size.